test/ex_aws_configurator/sqs_test.exs

```diff
@@ -13,6 +13,10 @@ defmodule ExAwsConfigurator.SQSTest do
     add_topic_to_config(build(:topic_config, name: :topic_name))
     add_queue_to_config(build(:queue_config, name: :raw_queue, raw_message_delivery: true))
 
+    add_queue_to_config(
+      build(:queue_config, name: :fifo_queue, content_based_deduplication: true, fifo_queue: true)
```

Comment:
não precisa terminar com `.fifo`?

Reply:
acho que sim, to fazendo uns testes aqui sem e nao rolou, mas setando o nome `.fifo` ta quebrando na hora da criacao por parametro invalido. To dando uma investigada